### PR TITLE
chore: Changelog v2.3.0

### PR DESCRIPTION
**Pull request by premanshup**

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,4 +1,19 @@
-v2.1.1
+v2.3.0
+- New: New options for Header Button - Open link in New Tab, add rel attribute. ( Doc Link )
+- Improvement: Hardened the security of plugin.
```
Comment (Swapnildhanrale) [inline changelog.txt]: @premanshup need to make it theme for Theme changelog

File: changelog.txt [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+- Improvement: Hardened the security of theme.
+- Improvement: Compatibility with latest WordPress PHP_CodeSniffer rules.
+- Improvement: Filter to remove featured image link on blog page.
+- Improvement: Fallback for batch process migration.
+- Fix: Protect theme button padding to be overridden by default values during update.
+- Fix: Theme minimum version update notice.
+- Fix: Breadcrumb customizer doc link fix. ( Doc Link )
+- Fix: W3C HTML validation header button ( Replaced button tag with div tag ).
+- Fix: Archive Description <p> tag missing - conflict with SEOPress.
+- Fix: Duplicate search with sticky header on mobile devices.
```
Comment (Balachandark) [inline changelog.txt]: This issue is not specific to mobile

File: changelog.txt [hunk 3]
```diff
@@ -1,4 +1,19 @@
-v2.1.1
+v2.3.0
+- New: New options for Header Button - Open link in New Tab, add rel attribute. ( Doc Link )
```
Comment (Balachandark) [inline changelog.txt]: New is added twice

File: changelog.txt [hunk 4]
```diff
@@ -1,4 +1,19 @@
...
+- Improvement: Hardened the security of theme.
+- Improvement: Compatibility with latest WordPress PHP_CodeSniffer rules.
+- Improvement: Filter to remove featured image link on blog page.
+- Improvement: Fallback for batch process migration.
+- Fix: Protect theme button padding to be overridden by default values during update.
+- Fix: Theme minimum version update notice.
```
Comment (Balachandark) [inline changelog.txt]: This issue was from addon so it should be removed from theme I think
@premanshup Your thoughts?

File: changelog.txt [hunk 5]
```diff
@@ -1,4 +1,19 @@
-v2.1.1
+v2.3.0
+- New: New options for Header Button - Open link in New Tab, add rel attribute. ( Doc Link )
+- Improvement: Hardened the security of theme.
+- Improvement: Compatibility with latest WordPress PHP_CodeSniffer rules.
+- Improvement: Filter to remove featured image link on blog page.
+- Improvement: Fallback for batch process migration.
```
Comment (Balachandark) [inline changelog.txt]: this should be `Added Fallback for database migration when CRON is disabled`

**Pull request merged**